plugins/MauticFocusBundle/Tests/EventListener/ReportSubscriberTest.php

```diff
@@ -0,0 +1,186 @@
+<?php
+
+namespace MauticPlugin\MauticFocusBundle\Tests\EventListener;
+
+use Mautic\ReportBundle\Event\ReportBuilderEvent;
+use Mautic\ReportBundle\Event\ReportGeneratorEvent;
+use MauticPlugin\MauticFocusBundle\EventListener\ReportSubscriber;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+class ReportSubscriberTest extends TestCase
+{
+    /**
+     * @var MockObject|ReportBuilderEvent
```

Comment:
It's a MockObject as well as the ReportbBuilderEvent. Static analysis tools and many IDEs will be more helpful with this small change. But I'd advise to avoid using mocks completely.
```suggestion
     * @var MockObject&ReportBuilderEvent
```

Reply:
Thanks for your feedback @escopecz  
All your points are now implemented. However there's a problem when running the CI workflow. All the tests with mariadb seem to be working, however tests with mysql are not working. I didn't manage to solve the problem yet. Do you have any ideas on how to fix it?